libs/langchain/langchain/schema/runnable/configurable.py

```diff
@@ -239,16 +239,15 @@ class RunnableConfigurableAlternatives(DynamicRunnable[Input, Output]):
 
     @property
     def config_specs(self) -> Sequence[ConfigurableFieldSpec]:
-        alt_keys = self.alternatives.keys()
-        which_keys = tuple(Literal[k] for k in alt_keys) + (  # type: ignore
-            Literal["default"],
+        which_enum = enum.StrEnum(  # type: ignore[misc]
+            self.which.name, list(self.alternatives.keys()) + ["default"]
```

Comment:
Should we worry about this note from the docs on `StrEnum`?
![image](https://github.com/langchain-ai/langchain/assets/2348618/36f9c571-d162-4cd3-b03d-0455575d795b)

Reply:
It's fine, we only check these values in this component